app/helpers/course_summaries_helper.rb

```diff
@@ -0,0 +1,69 @@
+module CourseSummariesHelper
+	###
+	#  Get JSON data for the table in the form of
+	#  { id: student_id, user_name: student.user_name,
+	#    first_name: student.first_name, last_name: student.last_name
+	#    marks: [ // an array of marks for each assignment, null if no grade ] }
+	###
+	def get_table_json_data
+		all_students = Student.where(type: 'Student');
```

Comment:
Use 2 (not 1) spaces for indentation.<br>Do not use semicolons to terminate expressions.<br>Tab detected.